packages/custom-functions-metadata-plugin/package-lock.json

```diff
@@ -1458,371 +1420,23 @@
       "dev": true
     },
     "eslint-plugin-office-addins": {
-      "version": "0.2.2",
-      "resolved": "https://registry.npmjs.org/eslint-plugin-office-addins/-/eslint-plugin-office-addins-0.2.2.tgz",
-      "integrity": "sha512-sinVHbsUUfOOiVRmcPeH+U1v7C0LBNFZKD8GczOR0nfHlFo81A0u5rNw8lMUJduDFhlJeyZmBH1n87rbID96LQ==",
+      "version": "0.3.0",
```

Comment:
Just a comment that packages which are using version 0.x, if they are ready, it might be good to bump them up to 1.0.0. With semver, if the major version is zero (first number), then the minor version (second number) is treated such that it won't automatically pick up minor version number updates. When being developed, it's therefore better to start with 0.x, but once it is ready for production, it probably is better to move to 1.x.

Reply:
No changes needed as part of this pr (in case it wasn't clear)...